src/listr/tasks/startNodeTaskFactory.js

```diff
@@ -58,7 +70,7 @@ function startNodeTaskFactory(dockerCompose) {
 
           if (driveImageBuildPath || dapiImageBuildPath) {
             if (config.get('network') === NETWORKS.TESTNET) {
-              this.error('You can\'t use drive-image-build-path and dapi-image-build-path options with testnet network', { exit: true });
+              throw new Error('You can\'t use drive-image-build-path and dapi-image-build-path options with testnet network', { exit: true });
```

Comment:
`Error` constructor accepts `filename` as the second param. You don't need to pass anything except `message` (the first argument)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error

Reply:
Fixed, is there a way to see the parameters required by a function in the IDE? I can't get this working in VS Code because using awilix seems to break the "Go to definition" function for all injected functions.

Reply:
You can use "go to function" feature by the function name.